File: Producto/routes-service/src/main/java/com/tagok/routes_service/dto/response/route/CobroTramoResponse.java
package com.tagok.routes_service.dto.response.route;

import java.math.BigDecimal;
import java.time.LocalDateTime;

import com.tagok.routes_service.domain.tarifa.TipoTarifa;

public record CobroTramoResponse(
    Long entradaId,
    Long salidaId,
    String codigoEntrada,
    String codigoSalida,
    String nombreEntrada,
    String nombreSalida,
    String autopista,
    double latitudEntrada,
    double longitudEntrada,
    double latitudSalida,
    double longitudSalida,
    TipoTarifa tarifa,
    BigDecimal valor,
    LocalDateTime fechaHora) implements CobroRutaResponse 
{

}

File: Producto/tag-ok-app/app/src/main/java/com/tagok/app/data/dto/route/CobroTramoResponse.kt
package com.tagok.app.data.dto.route

import kotlinx.datetime.LocalDateTime
import kotlinx.serialization.Serializable

@Serializable
data class CobroTramoResponse(
    val entradaId: Long,
    val salidaId: Long,
    val nombreEntrada: String,
    val nombreSalida: String,
    val autopista: String,
    val latitudEntrada: Double,
    val longitudEntrada: Double,
    val latitudSalida: Double,
    val longitudSalida: Double,
    val tarifa: String,
    val valor: Double,
    val fechaHora: LocalDateTime) : CobroRutaResponse()